Client/TeamTalkClassic/settings/ClientXML.h

```diff
@@ -26,11 +26,11 @@
 
 #include "Settings.h"
 
-#define TEAMTALK_XML_VERSION                        "5.4"
+#define TEAMTALK_XML_VERSION                        "5.6"
 
 #define TEAMTALK_XML_VERSION_DEFAULT                "5.0"
 
-#define DEFAULT_NICKNAME                            _T("NoName")
+#define DEFAULT_NICKNAME                            LoadText(IDS_DEFAULTNICKNAME, _T("NoName"))

```

Comment:
Put LoadText() into the .cpp file instead.

Then call like this:
LoadText(IDS_DEFAULTNICKNAME, DEFAULT_NICKNAME).

Please change TEAMTALK_XML_VERSION to "5.4" again.